src/docs/sphinx/dev_guide/ubuntu_build.rst

```diff
@@ -0,0 +1,192 @@
+.. ## Copyright (c) Lawrence Livermore National Security, LLC and
+.. ## other Serac Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+.. _ubuntu_build-label:
+
+==================
+Serac Ubuntu Build
+==================
+
+------------------
+Basic System Setup
+------------------
+
+We recommend installing some basic system-level development packages to minimize the
+amount of packages that Spack will build.
+
+Install clang version 14 and make it the default compiler:
+
+.. code-block:: bash
+
+    sudo apt install -y --no-install-recommends clang-14 clang-format-14 llvm-14 libomp-14-dev gfortran-13
+    # Set clang-14 as the default clang
+    sudo update-alternatives --install /usr/bin/clang++ clang++ /usr/bin/clang++-14 100 \
+    && sudo update-alternatives --install /usr/bin/clang clang /usr/bin/clang-14 100 \
+    && sudo update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-14 100
+
+Install required build packages to minimize what Spack will build:
+
+.. code-block:: bash
+
+    sudo apt-get -qq install -y --no-install-recommends build-essential bzip2 cmake libopenblas-dev \
+    lua5.2 lua5.2-dev openmpi-bin libopenmpi-dev unzip
+
+Optionally you can install packages to generate documentation:
+
+.. code-block:: bash
+
+    sudo apt-get -qq install -y --no-install-recommends graphviz python3-sphinx texlive-full doxygen
```

Comment:
Maybe a warning that `texlive` will use a lot of disk space?  Especially since we're installing `-y`